cmd/gazelle/fix_test.go

```diff
@@ -310,3 +310,29 @@ go_library(
 		})
 	}
 }
+
+func TestFix_LangFilter(t *testing.T) {
+	fixture := []testtools.FileSpec{
+		{
+			Path:    "BUILD.bazel",
+			Content: ``,
```

Comment:
Let's change this to:

```
load("@io_bazel_rules_go//go:def.bzl", "go_library")

go_binary(
    name = "nofix",
    library = ":go_default_library",
    visibility = ["//visibility:public"],
)

go_library(
    name = "go_default_library",
    srcs = ["main.go"],
    importpath = "example.com/repo",
    visibility = ["//visibility:public"],
)
```

Normally, `gazelle fix` would change `library = ":go_default_library"` to `embed = [":go_default_library"]` in the `go_binary` rule. But if the `go` extension is disabled, that shouldn't happen.

Reply:
SG, updated!